### DISCUSSION TITLE
Marketplace extension request (idea) - Email support ticket

### DISCUSSION THREAD

**Discussion by nordcart** [directus/directus#8419] [Feature Requests]
While we are waiting for the Directus Marketplace, I thought I could share an Idea :) .

I´m not a coder my self, but if someone think they could manage to build this extension and sell it on the marketplace it would be cool!

1. Connect an email to directus (send/recieve)
2. New email --> Move stuff into fields --> "Status NEW" - Subject "Subject" - Customer "email" etc.. (like in picture below)
3. Open the new ticket --> Answer (like a chat) --> this send an email to the customer with the answer --> change status

![image](https://user-images.githubusercontent.com/12813484/135257017-67b72490-0709-4d3c-a16c-df8231addc3d.png)

**Comment by nordcart** [comment]
Another idea would be to have a mailing list, from where you can send an email to all customers.

**Reply by benhaynes** [reply]
I like it. We already have the transactional email endpoints, so really this is about exposing a configurable UI for this... perhaps as an info sidebar component. 🤔 

As for the larger "Support Ticket" request, since we have hooks, this too is just about finding a UI that works for the underlying code. The difficulty with these is doing this in an unopinionated way.

**Reply by nordcart** [reply]
Allright, so everything is already there, just need to put together. I think this could be very useful in many projects. Let´s hope someone creates this 😃

**Comment by koksikus** [comment]
Hey, how did you get the effect of those colored buttons in the status? Through [conditions](https://docs.directus.io/reference/filter-rules/)? Can you reveal at least part of the secret? It looks amazing and I wish I had something like that too. Thx

**Reply by nordcart** [reply]
It´s no secret. That´s a screenshot from Google. But I colored labels in my project as well, look at the screenshots:
![image](https://user-images.githubusercontent.com/12813484/135582248-d67104f6-286c-47e7-9c3a-b6bfaca8d098.png)


If the value = kirjanpito --> the color is green
![image](https://user-images.githubusercontent.com/12813484/135582278-810fca13-74bd-444a-aeda-19a212c939a6.png)

**Reply by koksikus** [reply]
Thanks a lot!!! And the world becomes more beautiful.

How to achieve such effects.

**Reply by nordcart** [reply]
Trial and error, that´s how you learn 😃 . 

Guides: https://docs.directus.io/getting-started/introduction/

Or use discord: https://discord.gg/directus

**Reply by koksikus** [reply]
Hey, excuse me, can you please send me the invitation again for discord (time has passed to accept the invitation). Sorry!!

**Reply by benhaynes** [reply]
https://directus.chat

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
